linera-execution/src/policy.rs

```diff
@@ -41,6 +41,8 @@ pub struct ResourceControlPolicy {
     pub message: Amount,
     /// The additional price for each byte in the argument of a user message.
     pub message_byte: Amount,
+    /// The price for a performing an HTTP request.
+    pub http_request: Amount,
```

Comment:
```suggestion
    pub per_http_request_price: Amount,
```

We should also think about having a separate type for prices. And start calling it weights. See https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/revive/src/weights.rs for reference.

Reply:
That breaks the convention (see the fields above). But I agree we could improve the names. Feel free to open an issue to track this.